src/Symfony/Component/OptionsResolver/Debug/OptionsResolverIntrospector.php

```diff
@@ -101,4 +101,14 @@ public function getDeprecation(string $option): array
     {
         return ($this->get)('deprecated', $option, \sprintf('No deprecation was set for the "%s" option.', $option));
     }
+
+    /**
+     * @return \Closure[]
+     *
+     * @throws NoConfigurationException when no nested is configured
```

Comment:
what's a "nested"? "nested option"? "option"?

Reply:
well, the whole point of this component is handing options, everything is about options, I thought it could be implicit like `defined`, `required`, etc.

Reply:
we can be more explicit in this case though

Reply:
I updated this specific method and description